README.md

```diff
@@ -1,2 +1,18 @@
-# resolved
-A stub-resolver for Linux, macOS, and Windows
+# Host Resolver
+A stub DNS resolver that runs on the host machine on Linux, macOS, and Windows. The main goal behind this stub resolver is more robuset handling of domain name resolutions when VPN split tunnle is setup.  
+
+![b822615f-af15-4997-981b-53a6f1153d81 sketchpad (1)](https://user-images.githubusercontent.com/10409174/161135610-6418fe36-cde0-46d2-8e88-11cab9a3b3a2.svg)
```

Comment:
The URL shows a page with 2 black boxes to me. Does it work when rendered as part of Markdown?